Gruntfile.js

```diff
@@ -63,6 +63,7 @@ module.exports = function(grunt) {
      * LESS: https://github.com/gruntjs/grunt-contrib-less
      *
      * Compile LESS files to CSS.
+     * Source maps are slow, so they're separated into their own task for when needed
```

Comment:
Extremely minor, but if you're making edits in the PR, you could add a period at the end here.